Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1251,6 +1314,28 @@ def fetch_incidents_command():
             if updated_at > last_time:
                 last_time = updated_at
 
+    if demisto.params().get('fetch_pull_requests'):
```

Comment:
The integration should fetch either issues or pull requests, not both. Please separate the default issues fetch with an else block

Reply:
@DeanArbel ,will push another commit to reflect that